packages/appstore/src/components/modals/wallets-migration-failed/wallets-migration-failed.tsx

```diff
@@ -0,0 +1,115 @@
+                                        className='wallets-migration-failed__title'
+                                    >
+                                        {localize('Sorry for the interruption')}
+                                    </Text>
+                                    <Text size='xs'>
+                                        {localize(
+                                            "We're unable to complete with the Wallet upgrade. Please try again later or contact us via live chat."
+                                        )}
+                                    </Text>
+                                </Modal.Body>
+                                <Modal.Footer className='wallets-migration-failed__footer'>
+                                    <Button secondary large onClick={handLiveChatButtonClick}>
+                                        {localize('Go to live chat')}
+                                    </Button>
+                                    <Button primary large onClick={handleClose}>
+                                        {localize('Back to Trader’s Hub')}
+                                    </Button>
+                                </Modal.Footer>
+                            </div>
+                        </Modal>
+                    </DesktopWrapper>
+                    <MobileWrapper>
+                        <Modal
+                            is_open={is_wallet_migration_failed}
+                            toggleModal={handleClose}
+                            width='32.3rem'
+                            should_header_stick_body={false}
+                            has_close_icon={false}
+                        >
+                            <div ref={wallets_migration_failed_ref}>
+                                <Modal.Body className='wallets-migration-failed--mobile'>
+                                    <Text
+                                        as='h1'
+                                        size='xs'
+                                        color='prominent'
+                                        weight='bold'
+                                        className='wallets-migration-failed__title--mobile'
+                                    >
+                                        {localize('Sorry for the interruption')}
+                                    </Text>
+                                    <Text size='xxs'>
+                                        {localize(
+                                            "We're unable to complete with the Wallet upgrade. Please try again later or contact us via live chat."
+                                        )}
+                                    </Text>
+                                </Modal.Body>
+                                <Modal.Footer className='wallets-migration-failed__footer--mobile'>
+                                    <Button secondary large onClick={handLiveChatButtonClick}>
```

Comment:
Same here it doesn't show the Secondary Button.

Reply:
@aizad-deriv the same as above, I have 3 links in card description and the modals are equal